### DISCUSSION TITLE
Function (FN) key on macOS

### DISCUSSION THREAD

**Discussion by alloc33** [manna-harbour/miryoku#123] [Q&A] [answered]
I just started to use layout, is there any way to access macOS function key?

**Comment by manna-harbour** [comment, marked as answer]
Please use the Discussions Q&A category for questions: https://github.com/manna-harbour/miryoku/discussions/new?category=q-a.

In general, the Apple Fn key isn't supported by non-Apple keyboards. See https://github.com/zmkfirmware/zmk/issues/947 and https://github.com/qmk/qmk_firmware/issues/2179.

**Comment by manna-harbour** [comment]
From https://discord.com/channels/719497620560543766/720014518432038922/1007654291609227385:

> you can map a key to a code you never use (say, F18). then, you can use macos's hidutil to remap that key to fn in the os itself. This would not be natively implemented on ZMK of course (it would be rather impossible), but for those who want to type FN on macos with a zmk board, it is indeed possible. slight_smile

**Comment by GMBermeo** [comment]
> I just started to use layout, is there any way to access macOS function key?

Try to use `MO(0)` as `fn` key.

**Reply by robrecord** [reply]
This isn't the same thing.
